Header header
float64[] arc
float64[] total_arc
float64[] alpha
float64[] speed